braincube_connector/data/data.py

```diff
@@ -69,20 +69,41 @@ def _extract_format_data(raw_dataset: Dict[str, Any]) -> Dict[int, Any]:
         A formated dictionary {column_key: formated column data}
     """
     formatted_dataset = {}
+    parse_date = parameters.get_parameter("parse_date")
+
     for col in raw_dataset["datadefs"]:
         col_id = int(col["id"].split("/d")[1])
-        if col["type"] == "DATETIME" and parameters.get_parameter("parse_date"):
-            formatted_dataset[col_id] = _to_datetime(col[DATACOL])
+
+        if DATACOL not in col:
+            continue
```

Comment:
logguer un warning avec le nom de la dataDef (quelque chose comme warnings.warn("no data found for: "+ col["id"]))